tests/test_moe/test_moe_load_balance.py

```diff
@@ -176,6 +176,7 @@ def run_dist(rank, world_size, port):
     run_hybrid_zero_optim_test(rank, world_size, stage=2)
 
 
+@pytest.mark.skip(reason="moe need to be refactored")
```

Comment:
Note: This test cannot run since we no longer have `MoeModel` on the `feature/moe` branch.